libdnf/dnf-sack.c

```diff
@@ -1535,6 +1537,7 @@ dnf_sack_make_provides_ready(DnfSack *sack)
 
     if (priv->provides_ready)
         return;
+    repo_internalize_all_trigger(priv->pool);
```

Comment:
this seems a bit weird, because we call it before calling dnf_sack_make_provides_ready() from solve()